### PR TITLE
Implements the component schema-editor-bbox to define a bounding box

### PR DESCRIPTION
**Pull request by manuelroth**
This PR implements a component called `schema-editor-bbox` to define a bounding box:
- The tool can add a bbox component by defining the following schema:
```json
{
  "Q:type": "bbox",
  "type": "array",
  "title": "Sichtbarer Kartenausschnitt",
  "maxItems": 4,
  "items": {
    "type": "number"
  }
}
```
- The item in the database will have a property with an array containing the bbox values:
```json
"bbox": [
    3.2808756049348347,
    41.659012018672854,
    19.277405223180608,
    50.73716755982957
  ]
```
- The bounding box can be added by pressing Shift and dragging the mouse to create a rectangle

Screenshot 2019-07-26 at 17 42 10

### CODE REVIEW

File: client/src/elements/schema-editor/schema-editor-bbox.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<template>
+  <require from="./schema-editor-bbox.css"></require>
+  <div class="title-container">
+    <label class="input-group-title">
+      ${schema["title"] & toolT}
+    </label>
```
Comment (benib) [inline client/src/elements/schema-editor/schema-editor-bbox.html]: I would add an explanation here like: "<Shift> gedrückt halten und Rechteck mit der Maus aufziehen" or something to explain how it is to be used.

File: client/src/elements/schema-editor/schema-editor-geojson-point.js [hunk 2]
```diff
@@ -44,12 +44,12 @@ export class SchemaEditorGeojsonPoint {
       return;
     }
 
-    mapboxgl.accessToken = schemaEditorConfig.geojson.layer.accessToken;
```
Comment (benib) [inline client/src/elements/schema-editor/schema-editor-geojson-point.js]: can we keep the mapbox token support somehow? so users without their own tile service can still configure it?

File: client/src/elements/schema-editor/schema-editor-bbox.html (hunk 1 above)
Comment (manuelroth) [reply client/src/elements/schema-editor/schema-editor-bbox.html]: Implemented in https://github.com/nzzdev/Q-editor/pull/226/commits/cfeea81c494603d7eb347c1fda8e0fdc1abed466

File: client/src/elements/schema-editor/schema-editor-geojson-point.js (hunk 2 above)
Comment (manuelroth) [reply client/src/elements/schema-editor/schema-editor-geojson-point.js]: Fixed in https://github.com/nzzdev/Q-editor/pull/226/commits/f1b377d735840ed082e674d884587347cae614fb

**Pull request merged**